source/features/remove-is-open-is-closed.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+
+const countMatches = (string: string, regex: RegExp): number => {
+	return ((string || '').match(regex) ?? []).length;
+};
+
+function createMergeLink(): HTMLAnchorElement {
+	const linkMergedSearchParameters = new URLSearchParams(location.search);
+	const linkIsMerged: HTMLAnchorElement = (
+		<a href="" className="btn-link">
+		Merged
+		</a>
+	);
+	const regexpQueryTotal = /is:open|is:closed|is:issue/g;
+	const regexpQuery = /is:open|is:closed|is:issue/;
+
+	let linkMergedSearchString = new URLSearchParams(location.search).get('q') ?? select<HTMLInputElement>('#js-issues-search')!.value;
+
+	while (countMatches(linkMergedSearchString, regexpQueryTotal) > 1) {
+		linkMergedSearchString = linkMergedSearchString.replace(regexpQuery, '').trim();
+	}
+
+	if (countMatches(linkMergedSearchString, /is:merged/) === 1) {
+		linkMergedSearchParameters.set('q', linkMergedSearchString.replace(/is:merged/, 'is:issue').trim());
+		linkIsMerged.classList.add('selected');
+	} else if (countMatches(linkMergedSearchString, regexpQueryTotal) === 1) {
+		linkMergedSearchParameters.set('q', linkMergedSearchString.replace(regexpQuery, 'is:merged').trim());
+	}
+
+	linkIsMerged.search = String(linkMergedSearchParameters);
+
+	return linkIsMerged;
+}
+
+function init(): void {
+	const divTableListFiltersParent = select('div.table-list-filters');
+	const inputJsIssuesSearch = select<HTMLInputElement>('#js-issues-search');
+	if ((divTableListFiltersParent === null) || (inputJsIssuesSearch === null)) {
+		return;
+	}
+
+	const mergeLink = createMergeLink();
+
+	const containerTargetButtons = divTableListFiltersParent.children[0].children[0];
+	const targetButtons: HTMLCollectionOf<HTMLAnchorElement> = containerTargetButtons.children;
+	for (const link of targetButtons) {
+		link.firstElementChild!.remove();
```

Comment:
If you’re trying to remove the icon, it’s safer to run `select('.octicon', link)!.remove()` because the first element could change and we’d remove the wrong one.

Reply:
Didn't know about it. Thanks. It is done on my local repo.